libc/stdio_file.c

```diff
@@ -376,15 +379,15 @@ TEST_GROUP(stdio_line);
 TEST_SETUP(stdio_line)
 {
 	/* file preparation */
-	fd = fopen(stdpath, "w+");
-	TEST_ASSERT_NOT_NULL(fd);
+	filep = fopen(STDIO_TEST_FILENAME, "w+");
```

Comment:
- should be "w", we're not reading from the file here
- the file should be deleted in `TEST_TEAR_DOWN`